platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -59,15 +63,15 @@ public ExportCommand(ConfigurationMigrationService configurationMigrationService
         this(configurationMigrationService, security, pathBuilder.build());
     }
 
-    ExportCommand(ConfigurationMigrationService configurationMigrationService,
-            Security security, Path path) {
+    ExportCommand(ConfigurationMigrationService configurationMigrationService, Security security,
```

Comment:
Yep, no constructors in the command class. I like your -UnderTest class in the unit test:
```
    private class ExportCommandUnderTest extends ExportCommand {
        public ExportCommandUnderTest(ConfigurationMigrationService service, Path exportPath) {
            setService(service);
            setExportPath(exportPath);
        }
    }
```